internal/pkg/exec/dockerengine.go

```diff
@@ -264,6 +270,28 @@ func ValidatePlatform(platform *string) error {
 	return nil
 }
 
+// ValidateOS checks if the entered string is a Docker-buildable operating system.
+func ValidateOS(os *string) error {
+	if os == nil {
+		return nil
+	}
+	if aws.StringValue(os) != LinuxOS {
+		return fmt.Errorf("OS %s is invalid; %s: %s", aws.StringValue(os), english.PluralWord(len(validOperatingSystems), "the valid operating system is", "valid operating systems are"), english.WordSeries(validOperatingSystems, "and"))
+	}
```

Comment:
Hmm seems like there are some gaps between the actually validation (only check if it is linux OS) and description in error message.

Reply:
Right now Linux is the only OS supported by Fargate, but if more are supported in the future, we can add them here.

Reply:
I meant maybe we should loop through `validOperatingSystems`

Reply:
done!